ESMF/GOCART_GridComp/NI_GridComp/NI_GridCompMod.F90

```diff
@@ -1343,18 +1343,26 @@ subroutine NI_GridCompRun2_ ( gcNI, w_c, impChem, expChem, &
    if(associated(NI_pnh4aq%data2d)) NI_pnh4aq%data2d(:,:) = 0.
    if(associated(NI_pnh3aq%data2d)) NI_pnh3aq%data2d(:,:) = 0.
 
+if(mapl_am_i_root()) print*,'NI before thermo sum(NH3) = ',sum(w_c%qa(nNH3)%data3d)
```

Comment:
@gmao-esherman Does GEOS GOCART legacy run NI? If so, this will print a lot on each GOCART time step, will it not? Perhaps this should be pflogger based?

Reply:
GOCART Legacy does run NI. I  forgot to remove this print statement. I will remove this now.

Reply:
Thanks. And there are a lot of them! I just commented on the first.

Reply:
Yes, very good catch! I have removed all the prints.